backoffice/src/views/SingleLegacyBeaconRecordView.tsx

```diff
@@ -63,6 +64,22 @@ export const SingleLegacyBeaconRecordView: FunctionComponent<
 
   const hexId = beacon?.hexId || "";
   const numberOfUses = beacon?.uses?.length.toString() || "";
+
+  const deleteLegacyBeaconButton =
+    beacon.beaconStatus === BeaconStatuses.Deleted ? (
```

Comment:
This conditional rendering isn't working yet but I'm addressing it

Reply:
If it's deleted, you'll still need to read it to see the note?

Reply:
Exactly yeah so we still want to see the beacon record in this view, but if it's been deleted we don't wanna see the delete button as you can't re-delete a deleted record